www/src/scss/elm-theme.scss

```diff
@@ -0,0 +1,4 @@
+@use "~@edx/elm-theme/dist/core.css" as elmThemeCore;
```

Comment:
[inform] This will update in a future commit to the following:

```scss
@use "~paragon-style/scss/core/core" as paragonCore;
@use "~@edx/elm-theme/dist/core.css" as elmCore;
@use "~@edx/elm-theme/dist/light.css" as elmLight;
@import "base";
```